lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/aplinear/APLMeanSummarizer.java

```diff
@@ -29,6 +29,11 @@
         return Arrays.asList("count", "m1", "m2");
     }
 
+    @Override
+    public int[][] getEncoded(List<String[]> columns, DataFrame input) {
```

Comment:
why does this take in `input` if it's not being used? If the abstract method specifies that it should be an argument, we should re-define the abstract method?